lib/statesman/adapters/sequel.rb

```diff
@@ -0,0 +1,81 @@
+require_relative "../exceptions"
+
+module Statesman
+  module Adapters
+    class Sequel
+      attr_reader :transition_class, :parent_model, :observer
+
+      ::Sequel.extension(:inflector)
+
+      def initialize(transition_class, parent_model, observer)
+        @transition_class = transition_class
+        @parent_model = parent_model
+        @observer = observer
+      end
+
+      def create(from, to, metadata = {})
+        create_transition(from.to_s, to.to_s, metadata)
+      ensure
+        @last_transition = nil
+      end
+
+      def history
+        history_dataset.all
+      end
+
+      def last
```

Comment:
This method now accepts a `force_reload` keyword argument, this code can be refactored to support Ruby < 2.0.

```ruby
def last(force_reload: false)
  @last_transition = nil if force_reload
  @last_transition ||= history_dataset.last
end
```